api/src/main/java/org/pmiops/workbench/mandrill/MandrillConfig.java

```diff
@@ -0,0 +1,28 @@
+package org.pmiops.workbench.mandrill;
+
+import org.pmiops.workbench.config.WorkbenchConfig;
+import org.pmiops.workbench.mandrill.api.MandrillApi;
+import org.springframework.beans.factory.annotation.Qualifier;
+import org.springframework.context.annotation.Bean;
+import org.springframework.context.annotation.ScopedProxyMode;
+import org.springframework.web.context.annotation.RequestScope;
+
+@org.springframework.context.annotation.Configuration
+public class MandrillConfig{
+
+  private static final String MANDRILL_API_CLIENT = "mandrillApiClient";
+
+  @Bean(name=MANDRILL_API_CLIENT)
```

Comment:
Does it actually fail without this qualifier? My guess is that the places where we've done this are all cargo-culted from the FirecloudConfig, where we actually need two different bindings for the same class (each with different credentials).